lib/timeout/src/lib.rs

```diff
@@ -1,12 +1,11 @@
 extern crate futures;
 extern crate tokio_connect;
 extern crate tokio_timer;
-extern crate tower_service;
+extern crate tower_service as svc;
```

Comment:
This is just for consistency and namespacing.  Having stamped out many of these modules, it's way easier to understand if each type is named after its role.  E.g. `proxy::reconnect::{Layer, Stack, Service}` rather than `proxy::reconnect::{ReconnectLayer, ReconnectStack, ReconnectService}`, etc.  I find it way easier to keep track of what's going on if we always `use svc` and explicitly reference `svc::Service`, `svc::Stack`, etc